### ISSUE TITLE
Bottle builds failing to verify attestations due to missing gh token

### ISSUE THREAD

**Issue by scpeters** [osrf/homebrew-simulation#2815]
From the following failed build:

* [![Build Status](https://build.osrfoundation.org/buildStatus/icon?job=generic-release-homebrew_triggered_bottle_builder%2Flabel%3Dosx_ventura&build=1561)](https://build.osrfoundation.org/job/generic-release-homebrew_triggered_bottle_builder/label=osx_ventura/1561/) https://build.osrfoundation.org/job/generic-release-homebrew_triggered_bottle_builder/1561/label=osx_ventura/

~~~
==> Installing dependencies for osrf/simulation/gz-msgs10: abseil, cmake, pkg-config, osrf/simulation/gz-cmake3, eigen, osrf/simulation/gz-utils2, mpdecimal, ca-certificates, openssl@3, readline, sqlite, xz, python@3.12, libyaml, ruby, osrf/simulation/gz-math7, osrf/simulation/gz-tools2, protobuf and tinyxml2
==> Installing osrf/simulation/gz-msgs10 dependency: abseil
==> Downloading https://ghcr.io/v2/homebrew/core/abseil/manifests/20240722.0
Already downloaded: /Users/jenkins/Library/Caches/Homebrew/downloads/c028788fa2308b8803272259e36d1cb86d0530402d2615f560609775287c09da--abseil-20240722.0.bottle_manifest.json
==> Verifying attestation for abseil
Error: The bottle for abseil could not be verified.

This typically indicates a missing GitHub API token, which you
can resolve either by setting `HOMEBREW_GITHUB_API_TOKEN` or
by running:

  gh auth login
~~~

I believe this failure is due to https://github.com/Homebrew/homebrew-test-bot/pull/1230

**Comment by scpeters** [comment]
potential fix in https://github.com/Homebrew/homebrew-test-bot/pull/1238

**Comment by scpeters** [comment]
> potential fix in [Homebrew/homebrew-test-bot#1238](https://github.com/Homebrew/homebrew-test-bot/pull/1238)

this has been merged, so I'm rebuilding #2810 to test

* [![Build Status](https://build.osrfoundation.org/buildStatus/icon?job=generic-release-homebrew_triggered_bottle_builder&build=1562)](https://build.osrfoundation.org/job/generic-release-homebrew_triggered_bottle_builder/1562/) https://build.osrfoundation.org/job/generic-release-homebrew_triggered_bottle_builder/1562/

**Comment by scpeters** [comment]
it has gotten farther than the previous failure point, so I consider this fixed
